tools/testsys/src/install.rs

```diff
@@ -0,0 +1,37 @@
+use anyhow::{Context, Result};
+use clap::Parser;
+use model::test_manager::{ImageConfig, TestManager};
+
+/// The install subcommand is responsible for putting all of the necessary components for testsys in
+/// a k8s cluster.
+#[derive(Debug, Parser)]
+pub(crate) struct Install {
+    /// Controller image pull secret
+    #[clap(
+        long = "controller-image-secret",
+        short = 's',
+        env = "TESTSYS_IMAGES_PULL_SECRET"
+    )]
+    secret: Option<String>,
```

Comment:
We need to do a lot better with documentation.

also: (I think the short abbreviations are unhelpful unless they are very common parameters in a command that you will typing a lot. So for example `testsys status -r` is a great abbreviation, but here I think `-s` would just leave someone in the dark when reading a script etc)

```suggestion
    /// Controller image pull secret. This is the name of a Kubernetes secret that will be used to
    /// pull the container image from a private registry. For example, if you created a pull secret
    /// with `kubectl create secret docker-registry regcred` then you would pass
    `--controller-pull-secrest regcred`.
    #[clap(
        long = "controller-pull-secret",
        env = "TESTSYS_CONTROLLER_PULL_SECRET"
    )]
    secret: Option<String>,
```